Runtime/Pointer/ObjectPointer.cs

```diff
@@ -190,46 +152,26 @@ public enum Visibility
         public PointsRendererUnityEvent RenderDataChanged = new PointsRendererUnityEvent();
 
         /// <summary>
-        /// The activation state of the <see cref="ObjectPointer"/>.
+        /// Whether the <see cref="ObjectPointer"/> is currently activated.
         /// </summary>
-        public bool ActivationState
-        {
-            get;
-            protected set;
-        }
-
+        public bool IsActivated { get; protected set; }
+        /// <summary>
+        /// Whether the <see cref="ObjectPointer"/> is currently hovering over a target.
+        /// </summary>
+        /// <returns><see langword="true"/> if the <see cref="ObjectPointer"/> is currently hovering over a target, <see langword="false"/> otherwise.</returns>
+        public bool IsHovering { get; protected set; }
         /// <summary>
         /// Reports hover duration of the <see cref="ObjectPointer"/> over the current target.

```

Comment:
```suggestion
        /// How long the <see cref="ObjectPointer"/> is hovering over the current target.
```

Reply:
i've gone for `The duration that the <see cref="ObjectPointer"/> is hovering over the current target.`

Reply:
👍